docs/src/ToricVarieties/NormalToricVarieties.md

```diff
@@ -60,16 +60,16 @@ NormalToricVarietyFromGLSM(charges::fmpz_mat)
 ## Properties of Toric Varieties
 
 ```@docs
-hastorusfactor(v::AbstractNormalToricVariety)
-isaffine(v::AbstractNormalToricVariety)
-iscomplete(v::AbstractNormalToricVariety)
-isfano(v::AbstractNormalToricVariety)
-isgorenstein(v::AbstractNormalToricVariety)
-issimplicial(v::AbstractNormalToricVariety)
-issmooth(v::AbstractNormalToricVariety)
-isnormal(v::AbstractNormalToricVariety)
-isorbifold(v::AbstractNormalToricVariety)
-isprojective(v::AbstractNormalToricVariety)
+has_torusfactor(v::AbstractNormalToricVariety)
+is_affine(v::AbstractNormalToricVariety)
+is_complete(v::AbstractNormalToricVariety)
+is_fano(v::AbstractNormalToricVariety)
+is_gorenstein(v::AbstractNormalToricVariety)
+is_simplicial(v::AbstractNormalToricVariety)
+is_smooth(v::AbstractNormalToricVariety)
+is_normal(v::AbstractNormalToricVariety)
```

Comment:
But we have other places defining methods for `isnormal`, `istrivial`, `isprime`, and probably more. Should we really change this piecemeal, instead of all at once, and with a transition plan? With this PR we'll have both `is_normal` *and* `isnormal`, etc.

Reply:
I understand. Still, one has to start somewhere. So I wanted to bring this PR to your attention, in case you want to you use that as a starting point for a more general renaming PR during the upcoming week. (If not, no hard feelings at all. Just trying to simply things/cater to people's needs.)